src/vs/code/electron-main/windows.ts

```diff
@@ -968,25 +995,34 @@ export class WindowsManager implements IWindowsMainService {
 		return restoreWindows;
 	}
 
-	private parseFolderUriArg(arg: string): URI {
-		// Do not support if user has passed folder path on Windows
-		if (isWindows && /^([a-z])\:(.*)$/i.test(arg)) {
-			return null;
+	private argToUri(arg: string): URI {
+		try {
+			let uri = URI.parse(arg);
+			if (!uri.scheme) {
+				console.log(`Invalid URI string, scheme missing: ${arg}`);
```

Comment:
@aeschli please do not use `console.log` but our `ILogService` that we carefully added a while ago if you ever plan to access that data from a user.

Reply:
👍